src/config/typescript.js

```diff
@@ -93,6 +93,14 @@ export default {
             modifiers: ['requiresQuotes'],
             format: null,
           },
+          {
```

Comment:
Describe what this is doing, like all the above rules?
```suggestion
          // Ban the `I` prefix in interface names, e.g. `IProduct`
          {
```